src/adapter/src/coord/read_policy.rs

```diff
@@ -262,7 +265,7 @@ impl<S: Append + 'static> crate::coord::Coordinator<S> {
     pub(crate) async fn acquire_read_holds(&mut self, read_holds: &ReadHolds<mz_repr::Timestamp>) {
         // Update STORAGE read policies.
         let mut policy_changes = Vec::new();
-        let storage = self.controller.storage_mut();
+        let storage = &mut self.controller.storage;
```

Comment:
Here and below we could inline `storage` to `self.controller.storage` instead of aliasing it. What do you think about this?